mozfun/glean/timespan_nanos/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+description: Returns the number of nanoseconds represented by a Glean timespan struct.  See
```

Comment:
It would be nice to preserve newlines in these descriptions so that it ends up rendering with the link on a separate line.